src/videojs-hls.js

```diff
@@ -138,7 +138,14 @@ videojs.Hls.prototype.handleSourceOpen = function() {
     };
 
     oldMediaPlaylist = this.playlists.media();
-    this.setBandwidth(this.playlists);
+    // the bandwidth estimate for the first segment is based on round trip
+    // time for the master playlist. the master playlist is almost always
+    // tiny so the round trip time is dominated by latency and so the
+    // computed bandwidth is much lower than steady-state. to account for
+    // this, we scale the bandwidth estimate from the master playlist.
+    this.setBandwidth({
```

Comment:
It expects roundTripTime, bandwidth, and bytesRecieved. BytesReceived are defaulted to zero if not supplied. roundTripTime isn't actually used anywhere and setBandwidth() is happy to set it to undefined.